src/tools/admin/conseillers/desinscription-recrutes.js

```diff
@@ -0,0 +1,76 @@
+const CSVToJSON = require('csvtojson');
+const { program } = require('commander');
+
+program
+.option('-c, --csv <path>', 'CSV file path');
+
+program.parse(process.argv);
+
+const readCSV = async filePath => {
+  try {
+    // eslint-disable-next-line new-cap
+    const users = await CSVToJSON({ delimiter: ';' }).fromFile(filePath);
+    return users;
+  } catch (err) {
+    throw err;
+  }
+};
+
+const { execute } = require('../../utils');
+
+execute(__filename, async ({ db, logger, exit, Sentry }) => {
+
+  logger.info('Désinscription des conseillers déjà recrutés');
+  let promises = [];
+  await new Promise(resolve => {
+    readCSV(program.csv).then(async conseillers => {
+      const total = conseillers.length;
+      let count = 0;
+      let ok = 0;
+      let errors = 0;
+      if (total === 0) {
+        logger.info(`[DESINSCRIPTION COOP] Aucun conseiller dans le fichier fourni`);
+      }
+      conseillers.forEach(conseiller => {
+        let p = new Promise(async (resolve, reject) => {
+          const email = conseiller['email'].toLowerCase();
+          const conseillerCoop = await db.collection('conseillers').findOne({ email, statut: 'RECRUTE', estRecrute: true });
```

Comment:
il faudra également ajouter les index mongodb pour statut et estRecrute

Reply:
ok noté !

Reply:
statut était déjà présent donc j'ai ajouté estRecrutee et email